File: LeetCode/101-200/151. 反转字符串中的单词.java
import java.util.StringJoiner;
class Solution {
    public String reverseWords(String s) {
    String[] s1 = s.split(" ");
        int n = s1.length;
        StringJoiner sj = new StringJoiner(" ");
        for (int i = n - 1; i >= 0; i--) {
           String trim = s1[i].trim();
            if (trim.length() > 0) sj.add(trim);
        }
        return sj.toString().trim();
    }
}

File: LeetCode/101-200/151. 反转字符串中的单词.go
func reverseWords(s string) string {
	sb := &strings.Builder{}
	split := strings.Split(strings.Trim(s, " "), " ")
	n := len(split)
	for i := n - 1; i >= 0; i-- {
		if len(split[i]) >0 {
			sb.WriteString(split[i])
			if i > 0 {
				sb.WriteByte(' ')
			}
		}
	}
	return sb.String()
}